plenoptic/synthesize/synthesis.py

```diff
@@ -1717,7 +1762,7 @@ def _setup_synthesis_fig(self, fig, axes_idx, figsize,
             axes = fig.axes
         return fig, axes, axes_idx
 
-    def plot_synthesis_status(self, batch_idx=0, channel_idx=0, iteration=None,
+    def plot_synthesis_status(self, batch_idx=0, channel_idx=None, iteration=None,
```

Comment:
General comment: I wonder if when we do our refactoring of ``synthesis`` we can pull some of these plotting tools into like a ``synthesis_diagnostics.py`` or something.

Reply:
Yeah, agreed -- see #92 for this. That's what Kate and I had discussed so far, happy to hear any more ideas.

Reply:
The display code needs to be refactored along with the Synthesis refactor 👍